src/review/views.py

```diff
@@ -1360,7 +1361,10 @@ def create_completed_review_form(submission, review_id):
         data = json.loads(review_assignment.results.data)
 
         for relation in relations:
-            v = data[relation.element.name]
+            field_name = relation.element.name.encode(
```

Comment:
You're doing this over and over throughout this process. Wouldn't it be useful to create a single method that does this, limiting the possibility of getting `.encode('ascii', 'xmlcharrefreplace')` wrong with any of the points it's used?